src/lib/hooks.ts

```diff
@@ -0,0 +1,38 @@
+import { useDispatch, useMappedState } from "redux-react-hook";
+import { ActionCreator, bindActionCreators, SelectorSpec } from "../";
+import { Tree } from "../helpers/objects";
+import { applySpec } from "ramda";
+import { StoreContext } from "redux-react-hook";
+
+export const StoreProvider = StoreContext.Provider;
+
+/**
+ * Return store-bound action-creators.
+ *
+ * @param actions action-creators generated by "re-reduced/createActions"
+ */
+export function useActions<TActions extends Tree<ActionCreator<any>> = {}>(
+  actions: TActions
+): TActions {
+  return bindActionCreators(actions)(useDispatch()).actions;
+}
+
+export type SimpleMapStateToProps<TResult> = (state: any) => TResult;
+
+/**
+ * Return mappedState.
+ *
+ * @param selectorOrMapState either normal mapStateToProps (without ownProps as 2nd parameter) or selectorSpecs
+ */
+export function useReduxState<TResult, TState = any>(
+  selectorOrMapState:
+    | SelectorSpec<TResult, TState>
+    | SimpleMapStateToProps<TResult>
```

Comment:
SimpleMapStateToProps  needs to take in TState also

Reply:
Replaced `any` with 'TState` in SimpleMapStateToProps.